index.js

```diff
@@ -72,28 +82,41 @@ const getPasteByHashResponseBuilder = new GetPasteByHashResponseBuilder();
         searchPasteResponseBuilder,
         urlRepository,
         getPasteByHashResponseBuilder,
-        jwt,
+        authTokenService,
         loggerProvider
     });
     const urlRoutes = new UrlRouterBuilder({
         express,
         urlRepository,
         urlFactory,
         pasteRepository,
-        jwt,
+        authTokenService,
         createUrlResponseBuilder,
         idGenerator,
-        loggerProvider
+        loggerProvider,
+        pasteFactory
+    });
+    const authRoutes = new AuthRouterBuilder({
+        express,
+        authTokenService,
+        userFactory,
+        userRepository,
+        loggerProvider,
+        loginResponseBuilder,
+        registerResponseBuilder,
+        passwordHashService,
+        idGenerator
```

Comment:
Are u sure it is AuthRouter? Maybe it is UserRouter since it interacts with user?